### PR TITLE
MVI implementation

### PR DESCRIPTION
**Pull request by gurungnaresh495**
Mvi implementation using Resource

### CODE REVIEW

**Comment by myofficework000** [comment]
@nahCauhsoJ please review it

File: app/src/main/res/layout/fragment_random_dog.xml [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+    >
+
+    <!-- TODO: Update blank fragment layout -->
+    <TextView
+        android:layout_width="match_parent"
+        android:layout_height="match_parent"
+        android:text="@string/hello_blank_fragment"
+        />
+
+</FrameLayout>
```
Comment (gurungnaresh495) [inline app/src/main/res/layout/fragment_random_dog.xml:17]: remove this

Comment (nahCauhsoJ) [reply app/src/main/res/layout/fragment_random_dog.xml:17]: Yep. Other than the XML, it looks fine.

**Review by nahCauhsoJ — APPROVED** [review]
All good.

**Review by myofficework000 — APPROVED** [review]

**Pull request merged**